rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,476 @@
+             is_in: ["WXforYou"]
+           - variable: MetaData/stationIdentification
+             is_in: ["H0269","H0313","H0300","H0080","H0346","H0164","H0005","H0177","H0041","H0498","H0338","H0062","H0244","H0203","H0237","H0030","H0322","H0560","H0140","H0447"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["WYDOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["KBIT","KBRX","KHLD","KBFU","KCMS","KPER","KDRC"]
+
+         # Bounds Check
+         - filter: Bounds Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           minvalue: -130
+           maxvalue: 130
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     [ 5.0 ]
+               error_min:  [ 1.0 ]
+               error_max:  [ 5.0 ]
+               variable: windEastward
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windNorthward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     [ 5.0 ]
+               error_min:  [ 1.0 ]
+               error_max:  [ 5.0 ]
+               variable: windNorthward
+           action:
+             name: reject
+
+         # Time window filter
+         - filter: Domain Check
+           apply at iterations: 0,1
+           where:
+             - variable:
+                 name: MetaData/timeOffset
+               minvalue: -0.10
+               maxvalue: 0.10
+
+         #- filter: Temporal Thinning
+         #  apply at iterations: 0,1
+         #  min_spacing: PT00M
+         #  seed_time: *analysisDate
+         #  category_variable:
+         #    name: MetaData/stationIdentification
+
+         ## Print filter data
+         #- filter: Print Filter Data
+         #  message: Printing filter data
+         #  summary: true
+         #  variables:
+         #  - variable: MetaData/latitude
+         #  - variable: MetaData/longitude
+         #  - variable: MetaData/pressure
+         #  - variable: ObsType/windEastward
+         #  - variable: ObsValue/windEastward
+         #  - variable: ObsType/windNorthward
+         #  - variable: ObsValue/windNorthward
+         #  - variable: QCflagsData/windEastward
+         #  - variable: QCflagsData/windNorthward
+
```

Comment:
Why don’t we have a quality marker check specifically for wind observations? Is it because we already have an extensive acceptance list for specific providers and stations, making the quality marker check unnecessary? I apologize for asking so many questions; I'm working on similar YAML files for 187/287 based on yours, and I need to understand the reasoning behind your settings.

Reply:
Not too many questions. And definitely don't worry about it. I'm thinking, like you said, the mesonet use list is quite extensive. Perhaps non-mesonet winds will need to be handled differently. Perhaps, I should investigate further for the mesonet winds as well.

Reply:
I didn't notice any reject list applied based on the quality marker, except for surface pressure. Does this mean that for wind observations, the station/provider list includes only those with a quality marker between 0 and 2, while for air temperature and humidity, we also accept observations with a quality marker greater than 3?

Reply:
@spanNOAA : you can confirm these by looking at the geovals outputs.